BaseTools/BinWrappers/PosixLike/ProductDataFormatter

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash

```

Comment:
Will you update the commit message and PR title to ProductDataFormatter to keep with the current name?

Reply:
Done.